README.md

```diff
@@ -1,10 +1,40 @@
 # SierraUpdatePoller
```

Comment:
@kfriedman,

I added some description in the documentation will that suffice? What do you mean by deployment instructions?